hack/release.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+# release.sh: configurable signed-artefact release script
+# Copyright (C) 2016-2019 SUSE LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Im not sure we want to / we can have an Apache2 file in GPL2 repo

Reply:
I can dual-license it (or relicense it) because I'm the sole author -- but I would like to point out that `release.sh` is not a derived work (nor vice-versa) of `slirp4netns` so there's no legal basis GPL's copyright would apply to `release.sh`.

However, I can relicense it under MIT or MPLv2 and there won't even be any theoretical license problems.

Reply:
cc @siretart PTAL?

Reply:
technically I guess you could, it just makes everyone's live harder if you have to deal with a software package that is comprised of multiple licenses. I'd highly recommend to just stick to a single license in the package. Fortunately, it seems that @cyphar is OK with (dual) licensing as GPL-2.0+. I would not be surprised to see contributions that require the package to include extra licenses being rejected on the grounds of license simplicity. 

BTW, I usually license my packaging work under the exact same terms as the upstream package for precisely this reason.

Reply:
> I usually license my packaging work under the exact same terms as the upstream package for precisely this reason.

Just for the record, I'm not doing it to be frustrating -- the code was written for `umoci` (an Apache-2.0 licensed project) and was then used in `runc` (an Apache-2.0 licensed project) as well as some other similarly-licensed projects.

But sure, I will relicense it to MIT (to avoid dual-licensing drama) and re-post it.